ic-os/dev-tools/bare_metal_deployment/deploy.py

```
@@ -678,6 +740,18 @@ def main():
 
         sys.exit(0)
 
+    # Check that all important hostos metrics are available.
+    if args.check_hostos_metrics:
+        success = check_nodes_hostos_metrics(
+            bmc_infos=bmc_infos,
+            parallelism=args.parallel,
+        )
+
+        if not success:
+            sys.exit(1)
+
+        sys.exit(0)
```

When benchmark and check_hostos_metrics are both set to true, only benchmark will run.

Maybe this isn't an issue, I just don't want it to trip us up later. We could fail early if both are set to true. Up to you!

    if args.benchmark and args.check_hostos_metrics:
        log.error("Cannot run both benchmark and check_hostos_metrics at the same time. Please choose one.")
        sys.exit(1)